rosidl_generator_py/cmake/find_valid_typesupport.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Commonly the CMake filenames match the function / macro name for easier discoverability.